AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListView.swift

```diff
@@ -107,6 +121,48 @@ private struct SearchableItemListView: View {
         }
     }
 
+    /// The Password Manager download card definition.
+    private var itemListCardPasswordManagerInstall: some View {
+        ItemListCardView(
+            bodyText: Localizations.cardPasswordManagerInstallBody,
+            buttonText: Localizations.cardPasswordManagerInstallButtonTitle,
+            leftImage: {
+                Image(decorative: Asset.Images.bwLogo)
+                    .foregroundColor(Asset.Colors.primaryBitwardenLight.swiftUIColor)
+                    .frame(width: 24, height: 24)
+            },
+            titleText: Localizations.cardPasswordManagerInstallTitle,
+            actionTapped: {
+                openURL(ExternalLinksConstants.passwordManagerLink)
+            },
+            closeTapped: {
+                // https://livefront.atlassian.net/browse/BITAU-129
+            }
+        )
+        .padding(.top, 16)
+    }
+
+    /// The Password Manager sync card definition.
+    private var itemListCardSync: some View {
+        ItemListCardView(
+            bodyText: Localizations.cardPasswordManagerSyncBody,
+            buttonText: Localizations.cardPasswordManagerSyncButtonTitle,
+            leftImage: {
+                Image(decorative: Asset.Images.bwLogo)
+                    .foregroundColor(Asset.Colors.primaryBitwardenLight.swiftUIColor)
+                    .frame(width: 24, height: 24)
+            },
+            titleText: Localizations.cardPasswordManagerSyncButtonTitle,
+            actionTapped: {
+                openURL(ExternalLinksConstants.passwordManagerSettings)
+            },
+            closeTapped: {
+                // https://livefront.atlassian.net/browse/BITAU-129
```

Comment:
⛏️ Add TODO: so it's easy to detect if we need to check whether there are any todos in the code:
```suggestion
                // TODO: https://livefront.atlassian.net/browse/BITAU-129
```